app/Jobs/TileSingleImage.php

```diff
@@ -43,14 +57,17 @@ class TileSingleImage extends Job implements ShouldQueue
     /**
      * Create a new job instance.
      *
-     * @param Image $image The image to generate tiles for.
+     * @param Image $file The image to generate tiles for.
      *
      * @return void
      */
-    public function __construct(Image $image)
+    public function __construct(Image $file)
     {
-        $this->image = $image;
-        $this->tempPath = config('image.tiles.tmp_dir')."/{$image->uuid}";
+        $this->file = $file;
+        $this->tempPath = config('image.tiles.tmp_dir')."/{$file->uuid}";
+        // for uploadToStorage method
+        $this->storage = 'image.tiles.disk';
+        $this->fragment = fragment_uuid_path($file->uuid);
```

Comment:
This could also be a constructor argument (i.e. the "target path"). This way you can get around the fact that an `Image` or a `Biigle\FileCache\Contracts\File` is not guaranteed to have a uuid.

Reply:
resolved 0d84d7e911b06aa640fd65f1e8be3b0b13508625